app/views/measures/_measures.html.erb

```diff
@@ -143,13 +157,26 @@
   </div>
 
   <div id="export-measure-references">
-    <%= render partial: 'measures/measure_references',
-               collection: declarable.export_measures,
-               as: 'measure',
-               locals: {
-                 declarable: declarable,
-                 anchor: 'export',
-                 rules_of_origin_schemes: rules_of_origin_schemes || roo_all_schemes
-               } %>
+    <% if uk_declarable.presence&.export_measures&.any? %>
+        <%= render partial: 'measures/measure_references',
+                collection: uk_declarable.export_measures,
+                as: 'measure',
+                locals: {
+                    declarable: uk_declarable,
+                    anchor: 'export',
+                    rules_of_origin_schemes: rules_of_origin_schemes || roo_all_schemes
```

Comment:
we are using the same `rules_of_origin_schemes || roo_all_schemes` for both 
`uk_declarable.export_measures`
and
`xi_declarable.export_measures`

Not sure if it is correct.